packages/inter-protocol/test/reserve/reserve.test.js

```diff
@@ -77,9 +77,192 @@ test('reserve add collateral', async t => {
   );
 });
 
+/**
+ * @type {import('ava').Macro<
+ *   [
+ *     {
+ *       makeInvitationMaker: (
+ *         t: import('ava').Assertions,
+ *         powers: { zoe; reserve },
+ *       ) => {
+ *         makeInvitation: () => Promise<Invitation>;
+ *         failsAfterRevokeAll?: boolean;
+ *       };
+ *     },
+ *   ],
+ *   unknown
+ * >}
+ */
+const reserveWithCollateral = test.macro({
+  title(providedTitle = '') {
+    return providedTitle;
+  },
+  async exec(t, { makeInvitationMaker }) {
```

Comment:
There's no need to manually specify test title preservation: https://github.com/avajs/ava/blob/main/docs/01-writing-tests.md#reusing-test-logic-through-macros
```suggestion
const reserveWithCollateral = test.macro(async (t, { makeInvitationMaker }) => {
```

Reply:
Gotcha.